### PR TITLE
feat: popup dispatcher cleanup

### PR DESCRIPTION
**Pull request by ala-n**

### CODE REVIEW

**Review by YanaBr — APPROVED** [review]

File: src/modules/esl-base-popup/README.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ Authors: *Julia Murashko*, *Alexey Stsefanovich (ala'n)*.
 
 ESLBasePopup - is a custom element, that is used as a base for "Popup like" components creation.
 
+ESLPopupDispatcher - plugin component, that prevents activation of multiple ESLBasePopup instances in bounds of managed container.
+Usually (and by default) binds to document.body. Use native dom events to manage controlled instances state.
```
Comment (dshovchko) [inline src/modules/esl-base-popup/README.md:12]: proposal, may be "DOM events"?

**Review by dshovchko — APPROVED** [review]

File: src/modules/esl-base-popup/README.md (hunk 1 above)
Comment (ala-n) [reply src/modules/esl-base-popup/README.md:12]: Will be updated with the upcoming changes in the separate PR. 
Thanks!

**Pull request merged**